### ISSUE TITLE
error: object 'make_unlist_result_names' not found

### ISSUE THREAD

**Issue by yangming** [alyssafrazee/ballgown#138]
when I run one of the two command: 
`> gene_expression = gexpr(bg) `
or:
`> timecourse_res_gene = stattest(bg_zmaTfilt, feature="gene", meas='FPKM', covariate='time', timecourse=TRUE)`
and the two commands popped the same `error:`
```
Error in get(name, envir = asNamespace(pkg), inherits = FALSE) :
  object 'make_unlist_result_names' not found
```

Is there any workaround here? thanks

**Comment by JMF47** [comment]
Can you post your `sessionInfo()`? Smells like a package is out of date. Thanks!

**Comment by yangming** [comment]
Hi, JMF, here is my session info, hope it helps:

[sessioninfo.txt](https://github.com/alyssafrazee/ballgown/files/1874459/sessioninfo.txt)

**Comment by JMF47** [comment]
I'm more worried about the other packages that are out of date.
Could you try to make sure that all the packages under `imports` is updated to the latest version from BioC?

**Comment by yangming** [comment]
Hi, JMF, you are awesome! you have a powerful intuition. 
yes, one of packages is our of date. I only updated packages from my personal lib before, and this doesn't work. So I update all packages from system lib. It works well now. thank you for your hint.
I attached the lastest session info for your convenience.
[sessioninfo3.txt](https://github.com/alyssafrazee/ballgown/files/1875833/sessioninfo3.txt)
